cli/testdata/server-config.yaml.golden

```diff
@@ -0,0 +1,336 @@
+        # client.
+        # (default: <unset>, type: string)
+        clientCAFile: ""
+        # Policy the server will follow for TLS Client Authentication. Accepted values are
+        # "none", "request", "require-any", "verify-if-given", or "require-and-verify".
+        # (default: none, type: string)
+        clientAuth: none
+        # Paths to the private keys for each of the certificates. It requires a
+        # PEM-encoded file.
+        # (default: <unset>, type: string-array)
+        keyFiles: []
+        # Minimum supported version of TLS. Accepted values are "tls10", "tls11", "tls12"
+        # or "tls13".
+        # (default: tls12, type: string)
+        minVersion: tls12
+        # Path to certificate for client TLS authentication. It requires a PEM-encoded
+        # file.
+        # (default: <unset>, type: string)
+        clientCertFile: ""
+        # Path to key for client TLS authentication. It requires a PEM-encoded file.
+        # (default: <unset>, type: string)
+        clientKeyFile: ""
+        # Controls if the 'Strict-Transport-Security' header is set on all static file
+        # responses. This header should only be set if the server is accessed via HTTPS.
+        # This value is the MaxAge in seconds of the header.
+        # (default: 0, type: int)
+        strictTransportSecurity: 0
+        # Two optional fields can be set in the Strict-Transport-Security header;
+        # 'includeSubDomains' and 'preload'. The 'strict-transport-security' flag must be
+        # set to a non-zero value for these options to be used.
+        # (default: <unset>, type: string-array)
+        strictTransportSecurityOptions: []
+    # Most Coder deployments never have to think about DERP because all connections
+    #  between workspaces and users are peer-to-peer. However, when Coder cannot
+    # establish
+    #  a peer to peer connection, Coder uses a distributed relay network backed by
+    #  Tailscale and WireGuard.
+    derp:
+        # Whether to enable or disable the embedded DERP relay server.
+        # (default: true, type: bool)
+        enable: true
+        # Region ID to use for the embedded DERP server.
+        # (default: 999, type: int)
+        regionID: 999
+        # Region code to use for the embedded DERP server.
+        # (default: coder, type: string)
+        regionCode: coder
+        # Region name that for the embedded DERP server.
+        # (default: Coder Embedded Relay, type: string)
+        regionName: Coder Embedded Relay
+        # Addresses for STUN servers to establish P2P connections. Set empty to disable
+        # P2P connections.
+        # (default: stun.l.google.com:19302, type: string-array)
+        stunAddresses:
+            - stun.l.google.com:19302
+        # An HTTP URL that is accessible by other replicas to relay DERP traffic. Required
+        # for high availability.
+        # (default: <unset>, type: url)
+        relayURL:
```

Comment:
It seems like url types are the only fields that are printed out empty when the default is unset, the others seem to use the zero value.

Reply:
Maybe the type would be better as `string(url)` so that it's clear you're inputting a string here? Unsure though.

Reply:
I think for the most part it's self-explanatory.